### PR TITLE
Speed fixes

### PR DESCRIPTION
**Pull request by epicfaace**
- Move some script tags to end of <body>
- Add "rel='preload'" to all <link> tags
- Add "async" to all <script> tags

Hopefully these changes don't have any unexpected consequences, but doing this has a significant impact on page load.

Old: (audits on stanforddaily.com)
image

New: (audits on http://stanforddaily2.staging.wpengine.com, where this branch is deployed):
image

### CODE REVIEW

**Comment by epicfaace** [comment]
Actually don't merge this yet, the script async changes seem to have broken the "submit a tip" functionality.

**Comment by epicfaace** [comment]
@hesyifei should be ready to merge now.

**Comment by epicfaace** [comment]
@hesyifei

File: wp-content/themes/thestanforddaily/functions.php [hunk 1]
```diff
@@ -212,3 +212,16 @@ function tsd_scripts() {
 foreach($image_size as $id => $image) {
 	add_image_size( $id, $image['width'], $image['height'], $image['crop'] );
 }
+
+// Ref: https://stackoverflow.com/a/54816082/2603230
+// TODO: Broken on FireFox.
+function add_rel_preload($html, $handle, $href, $media) {
```
Comment (hesyifei) [inline wp-content/themes/thestanforddaily/functions.php]: This script prevents CSS from loading in Firefox.

**Comment by hesyifei** [comment]
Moved to https://github.com/TheStanfordDaily/tsd-wp-theme/pull/3

**Pull request closed without merging**